packages/docs/src/routes/docs/cheat-sheet/index.mdx

```diff
@@ -299,7 +299,7 @@ export function DynamicBackground() {
   const [blue, setBlue] = useState(0);
   return (
     <div style={
-      'background': `rgb(${red}, ${green}, ${blue})`
+      'background': `rgb({red}, {green}, {blue})`
```

Comment:
Actually, you will need those `$` signs here. and also an extra pair of curly braces. like this 
```jsx
style={{
      'background': `rgb(${red}, ${green}, ${blue})`
    }}
```

Reply:
great call